examples/hitl/rearrange_v2/README.md

```diff
@@ -33,6 +47,11 @@ Headless server:
 python examples/hitl/rearrange_v2/rearrange_v2.py +experiment=headless_server
 ```
 
+Testing Habitat-LLM episodes using a headed server:
```

Comment:
This shouldn't require `habitat_hitl.networking.enable=True` (this is a recent fix).

Also, can you mention: this is testing Habitat-LLM episodes with only a user-controlled humanoid (no Spot).

Reply:
Done